source/js/components/petition/petition.jsx

```diff
@@ -575,12 +575,14 @@ export default class Petition extends React.Component {
                 {this.state.userTriedSubmitting && !this.refs.privacy.checked && <small className="has-danger">Please check this box if you want to proceed</small>}
               </label>
             </div>
-            <div className="my-2">
-              <label className="form-check-label">
-                <input disabled={disableFields} type="checkbox" className="form-check-input" id="PrivacyCheckbox" ref="newsletterSignup" />
-                <span className="h6-heading form-text">Yes, I want to receive email updates about Mozilla’s campaigns.</span>
-              </label>
-            </div>
+            { this.props.subscribed ? null :
```

Comment:
This also needs a change to L233 now, which is currently `newsletterSignup: !!(this.refs.newsletterSignup.checked),` - this will throw when we don't have that element on the page, because `this.refs.newsletterSignup` won't exist, and so trying to access `checked` will cause a TypeError.

`newsletterSignup: this.props.subscribed || !!(this.refs.newsletterSignup.checked),` should do the trick.

Reply:
Good catch, but I might fix it in a slightly different way.

This would attempt to sign them up again (I think), probably wouldn't fail because they email they use is probably already in the system (and if it's not, would be signed up unexpectedly(I have two emails and don't always pay attention to which I use for what)) which is at best redundant, but at worst unexpected.